homedecor_bedroom/init.lua

```diff
@@ -31,26 +33,29 @@ local kbed_cbox = {
 }
 
 
-local bed_on_rightclick = minetest.registered_nodes["beds:bed"].on_rightclick
+local bed_on_rightclick =
+	minetest.registered_nodes["beds:bed"] and minetest.registered_nodes["beds:bed"].on_rightclick or nil
```

Comment:
as this is already 2 lines:
```lua
local bed_def = minetest.registered_nodes["beds:bed"]
local bed_on_rightclick = bed and bed.on_rightclick
```

Reply:
https://github.com/mt-mods/homedecor_modpack/pull/28/commits/c99997986580f9f949ba3337ccca9d635f8ddf8a fixed